tests/test_extensions.py

```diff
@@ -0,0 +1,132 @@
+"""
+Unit tests for Extension Manager
+"""
+
+import sys
+import os
+import unittest
+import mock
+import shutil
+
+dir_of_executable = os.path.dirname(__file__)
```

Comment:
You can still use `py.test tests/test_interfaces.py` to only run a specific file.